packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -375,6 +347,41 @@ export const useChatMessageCommonStyles = makeStyles({
       }
     }
   },
+  bodyWithPlaceholderImage: {
+    '& img[src=""]': {
+      display: 'block',
+      position: 'relative',
+      marginBottom: '5px'
+    },
+    '& img[src=""]:after': {
+      backgroundColor: tokens.colorNeutralBackground1Selected,
+      content: `url("data:image/gif;base64,R0lGODlhAQABAAAAACw=")`,
+      backgroundSize: 'center',
+      position: 'absolute',
+      width: '100%',
+      height: '100%',
+      top: '0',
+      left: '0',
+      display: 'block'
+    },
+    '& .broken-image-wrapper': {
+      width: '12rem',
+      height: '12rem',
+      marginTop: '0.75rem',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      outline: `1px solid ${tokens.colorNeutralForegroundDisabled}`,
```

Comment:
This should be ...shorthands.outline:...
https://griffel.js.org/react/api/shorthands/
https://developer.mozilla.org/en-US/docs/Web/CSS/outline